lib/iris/coords.py

```diff
@@ -1109,6 +1109,11 @@ def cell(self, index):
                 point = self.units.num2date(point)
                 if bound is not None:
                     bound = self.units.num2date(bound)
+                    
+        else
+            warn_deprecated("the 'Future' object property "
```

Comment:
Where does `warn_deprecated` come from? From what version has this been deprecated?